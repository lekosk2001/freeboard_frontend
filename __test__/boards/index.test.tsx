import Home from "../../pages/index"
import { render } from "@testing-library/react"

it("테스트", () => {
    const result = render(<Home></Home>)
    expect(result.container).toMatchSnapshot();
})