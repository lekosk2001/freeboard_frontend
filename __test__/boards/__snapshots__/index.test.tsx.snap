// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`테스트 1`] = `
<div>
  <div>
    div
  </div>
</div>
`;
